edx_sga/sga.py

```diff
@@ -82,7 +82,7 @@ class StaffGradedAssignmentXBlock(StudioEditableXBlockMixin, ShowAnswerXBlockMix
 
     display_name = String(
         display_name=_("Display Name"),
-        default=_('Staff Graded Assignment'),
+        default='Student Assessment',
```

Comment:
Use `ugettext_lazy()` /  `_()`

Reply:
<img width="690" alt="image" src="https://user-images.githubusercontent.com/43375602/160112189-8a589d80-5736-4b15-b18d-9098fcbb342c.png">
this was the previous commit removed by Ganesh

Reply:
https://github.com/mitodl/edx-sga/blob/v0.18.0/edx_sga/sga.py#L91 - Actual repo

Its good to use translation

Reply:
Updated the code , vinay please check it